modulefiles/module_base.wcoss2.lua

```diff
@@ -0,0 +1,62 @@
+help([[
+Load environment to run GFS on WCOSS2
+]])
+
+PrgEnv_intel_ver=os.getenv("PrgEnv_intel_ver")
```

Comment:
Since these are each only being used once, do we want to just use os.getenv in-line in the load statements instead of loading them to variables first?

Reply:
Addressed.